src/components/site/guidelines/guidelines.tsx

```diff
@@ -0,0 +1,29 @@
+import { Guide } from "../guide";
+import styles from "./guidelines.module.css";
+
+const guidelinesInfo: string[] = [
+	"火気の取り扱いはご遠慮ください.",
+	"キャンパス内は全面喫煙となっております.",
+	"指定場所以外で飲食をすることはできません。飲食可能な場所についてはパンフレットをご確認ください。",
+	"構内の機材や設備の撮影はおやめください.",
+	"事故防止のため、構内の機材にはお手を触れないようにお願いいたします。",
+	"重要なアナウンスが流れる場合がございます。構内放送にご留意ください。",
+	"他のお客様への迷惑行為や、スタッフの指示に従わない等の行為が見受けられた場合、退場または入場をお断りする場合があります。",
+];
+
+export function Guidelines() {
+	const headingText: React.ReactNode = (
+		<>
+			キャンパス内での
+			<br className={styles.br} />
+			注意事項
```

Comment:
メディアクエリで改行を調整するのは推奨しない
代わりに`inline-flex`を用いる

```suggestion
			<span>キャンパス内での</span>
			<span>注意事項</span>
```

Reply:
直しました